src/renderers/ListRenderer.php

```diff
@@ -140,6 +140,7 @@ private function renderRowContent($index = null, $item = null)
             $content[] = $this->renderActionColumn($index);
         }
 
+        $this->cloneButton && $content[] = $this->renderCloneColumn();
```

Comment:
if be honest i don't like such construction. Why don't use a basic if statement? as for me it is more readable

Reply:
You are right, its bad code style. Fixed.